app/javascript/react/components/AlbumIndex.js

```diff
@@ -26,6 +27,7 @@ const AlbumIndex = (props) => {
     return (
       <AlbumTile
         key={album.id}
+        id={album.id}
         album={album.album}
         art={album.art}
         artist={album.artist}
```

Comment:
At this point I would again pass the whole album as `album={album}` and destructure it at the `AlbumTile`